### DISCUSSION TITLE
[Bug Report] 官方文档打开后台空置一段时间 切回来 就无法点击了，需要刷新页面才行

### DISCUSSION THREAD

**Discussion by feifanshijie** [youzan/vant#12437] [General]
### 重现链接

https://vant-ui.github.io/vant/#/zh-CN/field

### Vant 版本

官方文档

### 描述一下你遇到的问题。

https://vant-ui.github.io/vant

### 重现步骤

1、官方文档打开
2、后台空置一段时间 切回来 就无法点击了，需要刷新页面才行

**Comment by chenjiahan** [comment]
我没遇到过该问题，有其他人遇到类似的问题嘛？
